tasks/pdr-cleanup/pdr_cleanup.py

```diff
@@ -0,0 +1,97 @@
+"""Task for archiving PDRs."""
+
+import json
+import logging
+import os
+from datetime import datetime
+
+import boto3
+from cumulus_logger import CumulusLogger
+from run_cumulus_task import run_cumulus_task
+
+logger = CumulusLogger("pdr-cleanup", logging.INFO)
+
+
+def cleanup_pdr(event: dict, context: dict):
+    """Task to archive PDRs.
+
+    Args:
+        event (dict): A lambda event object
+        context (dict): An AWS Lambda context
+
+    """
+
+    logger.info("## EVENT OBJ \n" + json.dumps(event))
+
+    provider = event["config"]["provider"]
```

Comment:
Consider adding schemas either json or using Pydantic to better define expected input structures and types.  This would help better define the contracts for the task and the expectations of what to pass in the CMA structure as well as outputs.